packages/main/src/Carousel.js

```diff
@@ -347,17 +362,21 @@ class Carousel extends UI5Element {
 	}
 
 	_onmouseout() {
-		this._visibleNavigationArrows = false;
+		if (this.arrowsPlacement === CarouselArrowsPlacement.Content) {
+			this._visibleNavigationArrows = false;
+		}
 	}
 
 	_onmouseover() {
-		this._visibleNavigationArrows = true;
+		if (this.arrowsPlacement === CarouselArrowsPlacement.Content) {
+			this._visibleNavigationArrows = true;
```

Comment:
there is a property "hideNavigation", is it considered here?

Reply:
It was deleted by #3228

Reply:
Yes. The check for `hideNavigationArrows` happens during rendering. At this point in execution, it is established that `arrowsPlacement` will not change.